fastchat/model/model_adapter.py

```diff
@@ -221,6 +221,8 @@ def load_model(
                 device=device,
```

Comment:
You should remove the warning in L214.  Otherwise, your code is not executed.

Reply:
done